website/docs/zh/config/lib/dts.mdx

```diff
@@ -245,3 +250,48 @@ export default {
   ],
 };
 ```
+
+### dts.experiments
+
+- **类型：** `{ tsgo?: boolean }`
+- **默认值：** `{}`
+
+用于启用实验性功能。
+
+#### dts.experiments.tsgo
+
+- **类型：** `boolean`
+- **默认值：** `false`
+
+是否使用 [tsgo](https://github.com/microsoft/typescript-go) 生成类型声明文件。
+
+如果需要启用该选项，需要安装 [@typescript/native-preview](https://www.npmjs.com/package/@typescript/native-preview) 作为开发依赖。
+
+<PackageManagerTabs command="add @typescript/native-preview -D" />
+
+::: tip 版本要求
+
+`@typescript/native-preview` 要求 Node.js 20.6.0 或更高版本。
+
+:::
+
+```ts title="rslib.config.ts"
+export default {
+  lib: [
+    {
+      dts: {
+        // [!code highlight:3]
+        experiments: {
+          tsgo: true,
+        },
+      },
+    },
+  ],
+};
+```
+
+::: note
+
+`tsgo` 可以提供更快的类型声明文件生成速度，尤其是对于大型项目。然而，由于 `tsgo` 仍处于实验阶段，可能会存在一些未解决的问题或限制。因此，在启用该选项前，请确保在你的项目中进行充分测试。
```

Comment:
move the pros of `tsgo` to the top of this section, that's the very first reason for user to enable it. and list comprehensive cons of tsgo (unstable / mis-aligned version of ts-server, etc).